deploy/helm/templates/events-collector-deployment.yaml

```diff
@@ -34,7 +34,8 @@ spec:
     spec:
       terminationGracePeriodSeconds: {{ .Values.otel.events.terminationGracePeriodSeconds }}
       serviceAccountName: {{ include "common.fullname" . }}
-      securityContext: {}
+      securityContext: 
+        fsGroup: 10001
```

Comment:
How does this related to namespace filter?

Reply:
it is not, that is the fix for the storage issue